pytest-framework.ini

```diff
@@ -7,7 +7,6 @@ testpaths =
     src
 addopts = 
     -p pytester
-    --ignore=src/pytest_plugins/test_filler/test_filler.py
     --ignore=src/pytest_plugins/consume/direct/test_direct.py
     --ignore=src/pytest_plugins/consume/hive_simulators/engine/test_via_engine.py
     --ignore=src/pytest_plugins/consume/hive_simulators/rlp/test_via_rlp.py
```

Comment:
I actually wonder if we can just remove these too:
```suggestion
```

As we don't have the `__init__.py` files. Let me double check.

Reply:
Hmm nope :(

We could also just ignore all of `consume`:
```
--ignore=src/pytest_plugins/consume/
```

I don't see us adding any framework tests to consume any time soon!

Reply:
I think it's fine as is, it's only three extra lines, we can change it once we implement tests for consume IMO.